osidb/serializer.py

```diff
@@ -504,6 +505,41 @@ class Meta:
         abstract = True
 
 
+class JiraTaskSyncMixinSerializer(serializers.ModelSerializer):
```

Comment:
You don't seem to be using this mixin anywhere? If you did on the `FlawSerializer` you wouldn't need the check on `Flaw.create`, also this doesn't take into account the env variable you defined earlier